### PR TITLE
Move VRF contracts to a VRF folder.

### PR DESCRIPTION
**Pull request by vreff**

### CODE REVIEW

File: contracts/src/v0.8/tests/VRFConsumer.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.0;
 
 import "../interfaces/LinkTokenInterface.sol";
-import "../VRFConsumerBase.sol";
+import "../vrf/VRFConsumerBase.sol";
```
Comment (makramkd) [inline contracts/src/v0.8/tests/VRFConsumer.sol:5]: Worth grouping VRF contracts into a single sub-folder within `contracts/src/v0.8/tests/vrf`?

Comment (RensR) [reply contracts/src/v0.8/tests/VRFConsumer.sol:5]: +1

File: contracts/scripts/native_solc_compile_all [hunk 2]
```diff
@@ -53,36 +53,36 @@ $SCRIPTPATH/native_solc8_6_compile ChainlinkClient.sol
...
 # VRF V2
 $SCRIPTPATH/native_solc8_6_compile vrf/VRFConsumerBaseV2.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFConsumerV2.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFMaliciousConsumerV2.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFTestHelper.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFV2RevertingExample.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFV2ProxyAdmin.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFV2TransparentUpgradeableProxy.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFConsumerV2UpgradeableExample.sol
+$SCRIPTPATH/native_solc8_6_compile tests/vrf/VRFConsumerV2.sol
```
Comment (RensR) [inline contracts/scripts/native_solc_compile_all:62]: I thought we wanted these in `/vrf/testhelpers/...` to have them live inside the vrf folder

@makramkd correct?

Comment (vreff) [reply contracts/scripts/native_solc_compile_all:62]: Per comment `contracts/src/v0.8/tests/vrf`

Comment (vreff) [reply contracts/scripts/native_solc_compile_all:62]: I am completely indifferent here.

Comment (makramkd) [reply contracts/scripts/native_solc_compile_all:62]: Lets name them `testhelpers` because otherwise we'd think those are actual tests.

**Review by makramkd — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**